### ISSUE TITLE
ui5-list keyboard navigation should be vertical only

### ISSUE THREAD

**Issue by pskelin** [SAP/ui5-webcomponents#862]
**Describe the bug**
only the up/down arrows should change the focused list item, but now the left/right arrows also change it which should not be the case.

**To reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/test-resources/sap/ui/webcomponents/main/pages/List.html
2. focus a list item
3. use the left/right arrows

**Expected behavior**
left/right arrows should not change the focused item

**Context**
- UI5 Web Components version: 1.0.0-rc.4

**Affected components** *(if known)*
ui5-list
